conductor/justfile

```diff
@@ -86,7 +86,7 @@ watch:
 
 run-postgres:
 	docker rm --force pgmq-pg || true
-	docker run -d --name pgmq-pg -e POSTGRES_PASSWORD=postgres -p 5431:5432 quay.io/tembo/pgmq-pg:v0.14.2
+	docker run -d --name pgmq-pg -e POSTGRES_PASSWORD=postgres -p 5431:5432 quay.io/tembo/pg16-pgmq:latest
```

Comment:
Should this be version locked to a version of pgmq?

Reply:
Yeah I based this off what was in the pgmq repo, which is `latest`.  I updated it to pin the version to `v1.3.3`